src/neuroconv/datainterfaces/ecephys/openephys/openephysdatainterface.py

```diff
@@ -42,7 +42,8 @@ def __init__(
                 folder_path=folder_path, experiment_id=experiment_id, recording_id=recording_id, verbose=verbose
             )
             self.recording_extractor = OldToNewRecording(oldapi_recording_extractor=self.recording_extractor)
-
+            # Remove when spikeinterface 0.95 is released, this has an int sampling rate that causes problems
+            self.recording_extractor._sampling_frequency = float(self.recording_extractor.get_sampling_frequency())
```

Comment:
Shouldn't we use `recording_extractor.set_sampling_frequency()` instead?

Reply:
Just in case there's ever an extractor out there (or a custom extractor as well) that doesn't call it from the private attribute for some reason

Reply:
I could not find such a method in base recording.

Reply:
OK I must have misremembered or assumed it behaved more like an official property, my bad.